.github/workflows/ci.yml

```diff
@@ -65,4 +65,21 @@ jobs:
       run: |
         cd bioio-my-custom-file-format
         just update-from-cookiecutter
-        git status
\ No newline at end of file
+        git status
+
+  # Required by bioio-devs organization to succeed in order to merge
+  # done this way due to matrix testing strategy providing difficult
+  # to track workflow test names
+  results:
```

Comment:
It looks like it is just exiting with error code if test-repo-gen resulted in failure or cancellation?  Is this related to wanting only specific status checks to be required for PR merge as opposed to all?  I think it would help readability, at least for me, with extra punctuation.
```
  # Required by bioio-devs organization to succeed in order to merge.
  # Done this way due to matrix testing strategy providing difficult-to-track
  # workflow test names.
```

Reply:
It isn't so much that I want specific checks to be required rather than all so much as you can't select all as an option in GitHub (outside of literally specifying each individual test name which is impossible), but yes that is what this is about.